src/element.ts

```diff
@@ -0,0 +1,275 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
```

Comment:
Why does not it have diff?

Reply:
> @0xfe I have been investigating how to get the Element diff to work. It is possible doing the migration in two PRs:
> 
> 1. Sofware changes with all the interfaces in types/common
> 2. Modifications of Comments and movement of interfaces to the element.ts file
>    Should I proceed like that? Or should we consider as @wassertim said using another tool to look into the changes.

Reply:
@rvilarl Looks like you can use the `-M` flag to tell git what diff threshold to use for detecting renames. You can try it with a small value like `-M0.4` (<40% file change.)

Reply:
@0xfe is this one thing that we could configure in github as well or just locally

Reply:
@0xfe this is not working for me even locally, I suggest to follow the two PRs approach, is that fine?

Reply:
Yes, it's fine. Thanks for trying. I'd suggest calling out (in github comments) parts where the TS diverges a lot from the JS.

Reply:
Actually for the next PR, can we try the following to make the diffs work? 1) send PR with just rename of file, and add an eslint-diasble for the whole file, 2) send PR with actual changes and eslint-disable removed.

We can even try it for this PR if you're up for it.

Reply:
@0xfe This is a very good idea. I have also decided to go for the 2 steps approach because I believe two other changes where overseen due to the lack of diff. I will set up the questions directly on the code and also a question related on how to proceed on annotations,...